sdks/python/apache_beam/runners/interactive/pipeline_instrument.py

```diff
@@ -367,15 +375,13 @@ def cacheable_key(pcoll, pcolls_to_pcoll_id, pcoll_version_map=None):
 
 
 def has_unbounded_source(pipeline):
-  """Checks if a given pipeline has any source that is unbounded."""
+  """Checks if a given pipeline has replaceable unbounded sources."""
```

Comment:
I think we probably want to change this method to return a list of PTransforms that represent unbounded source since we will need them to be replaced by TestStream later. However, it's probably outside the scope of this PR.

Reply:
Yep, that's up next.